lib/tasks/import_datastream.rake

```diff
@@ -6,14 +6,18 @@
 namespace :ssg do
   desc 'Check if the latest SSG has been synced'
   task check_synced: [:environment] do
-    datastreams_message = "SSG datastreams not synced\nDatastream config revision: " \
-      "#{SupportedSsg.revision.inspect}\nDB revision: " \
-      "#{Revision.datastreams.inspect}"
-    remediations_message = "SSG remediations not synced\nRemediation config revision: " \
-      "#{SupportedRemediations.revision.inspect}\nDB revision: " \
-      "#{Revision.remediations.inspect}"
-    abort datastreams_message if Revision.datastreams != SupportedSsg.revision
-    abort remediations_message if Revision.remediations != SupportedRemediations.revision
+    if Revision.datastreams != SupportedSsg.revision(true)
```

Comment:
The force is very subtle here. If one would remove the first condition the second one would have again cached values. Maybe add a comment?

Reply:
This is the only place this check is actually needed, if someone removes it, things will go bad on many other levels.